src/vorta/views/repo_add_dialog.py

```diff
@@ -242,10 +250,10 @@ def run(self):
             params = BorgInfoRepoJob.prepare(self.values)
             if params['ok']:
                 self.saveButton.setEnabled(False)
-                thread = BorgInfoRepoJob(params['cmd'], params)
-                thread.updated.connect(self._set_status)
-                thread.result.connect(self.run_result)
-                self.thread = thread  # Needs to be connected to self for tests to work.
-                self.thread.run()
+                job = BorgInfoRepoJob(params['cmd'], params)
```

Comment:
This will be queued in the `default` queue rather than the repo specific queue.

Reply:
👍🏼

Reply:
But I guess the `BorgInfoRepoJob` is used to validate existing repo right ?

Reply:
Oh! you mean the BorgInit job?

Reply:
Yes, that's right. You have to pass an extra argument iirc.

Reply:
> 

Hi! sorry for being inactive, I was sick 😅 

I see the BorgJob accepting 3 params, 2 are already there, the third one should be sending value `default` by default since it is not provided 🤔 I was wondering how it's deciding about queue (this question might sound dumb but 😅 )

Reply:
> Yes, that's right. You have to pass an extra argument iirc.

Hi! sorry for being inactive, I was sick 😅 

I see the BorgJob accepting 3 params, 2 are already there, the third one should be sending value `default` by default since it is not provided 🤔 I was wondering how it's deciding about queue (this question might sound dumb but 😅 )